processor/core/src/main/java/com/redhat/service/smartevents/processor/actions/input/InputAction.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.smartevents.processor.actions.input;
+
+import com.redhat.service.smartevents.processor.actions.ActionBean;
+
+public interface InputAction extends ActionBean {
```

Comment:
Some suggestions on other names perhaps to make the purpose of this `Action` clearer:

- `SourceAction`
- `IngestAction`

Just thinking out loud as `InputAction` is a little bit hazy as to it's intent.

Reply:
I honestly thought about `SourceAction` as first name but then I thought also it would have caused confusion, but honestly it's the clearest once you understand how it works. :smile: